charts/portal/README.md

```diff
@@ -4,6 +4,9 @@ The Layer7 API Developer Portal (API Portal) is part of the Layer7 API Managemen
 ## Introduction
 This Chart deploys the Layer7 API Developer Portal on a Kubernetes Cluster using the Helm Package Manager.
 
+## 2.3.13 General Updates
```

Comment:
@melil02 Thanks for making the changes. We are thinking of merging stable to **develop/portal** and then making the 5.2.1 changes as **stable** and **develop/portal** branches are not in sync. Stable has a 2.3.2 chart version, so the next version should be **2.3.3** instead of **2.3.13**. Also there are documentation links that are required to update the version.